### ISSUE TITLE
Using multiple databases when creating an html document with interactive sql examples?

### ISSUE THREAD

**Issue by shafayetShafee** [nalgeon/sqlime#16]
Hey thanks for making [this](https://sqlime.org/examples.html) awesome javascript widget to create interactive sql examples.

My question whether it is possible to use multiple databases when creating an html document with interactive sql examples? That is, would the following work,

````
<pre class="employees">select
  rank() over w as "rank",
  name, department, salary
from employees
window w as (order by salary desc)
order by "rank", id;</pre>

<pre class="demo">select * from employees limit 10;</pre>

<sqlime-db name="employees" path="./employees.sql"></sqlime-db>
<sqlime-examples db="employees" selector="pre.employees" editable></sqlime-examples>

<sqlime-db name="demo" path="./demo.db"></sqlime-db>
<sqlime-examples db="demo" selector="pre.demo" editable></sqlime-examples>
````

*Note: both `employees.sql` and `demo.db` are taken from this repo*

**Comment by nalgeon** [comment]
Sure! You can connect as many databases as you want - just create a separate `sqlime-db` for each of them.

**Comment by shafayetShafee** [comment]
Thanks for your prompt reply! But I am having multiple issues with creating a static html file with interactive SQL examples. So I would really appreciate your help. I am posting these problems here in this thread.  But if it helps you to track the issues separately I can post separate ones.

## [Problem One] Failed to load database from URL

Consider the following reprex,

``` html
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>sqlime</title>
	<script src="https://unpkg.com/@antonz/sqlite@3.40.0/dist/sqlite3.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-db.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-examples.js"></script>
</head>
<body>

<pre class="employees">select
  rank() over w as "rank",
  name, department, salary
from employees
window w as (order by salary desc)
order by "rank", id;</pre>

<sqlime-db name="employees" path="./employees.sql"></sqlime-db>
<sqlime-examples db="employees" selector="pre.employees" editable></sqlime-examples>

</body>
</html>
```

Now If I open this html file on my browser (Google Chrome), I get the following,

![interactive example with failed to load error when clicking on run button](https://github.com/nalgeon/sqlime/assets/48413872/126fb778-c049-42cf-aef0-6fc915881339)

[*Note: If you are wondering about the wiggly lines under `desc` it just due to Grammarly extension enable in my browser*]

And I am getting these errors on the Chrome DevTools console,

![errors on chrome devtools console](https://github.com/nalgeon/sqlime/assets/48413872/f5434dcc-faff-4551-97be-a8e0dfdb1da4)

**Comment by nalgeon** [comment]
For JS to work, you must [serve the page](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server) through a web server, not just open it from the file system.

**Comment by shafayetShafee** [comment]
Ok, I get it. Thanks for letting me know!

**Comment by shafayetShafee** [comment]
But can you help with the second problem? Using multiple databases is leading to the errors.

For example, consider the following reprex,

``` html
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>sqlime</title>
	<script src="https://unpkg.com/@antonz/sqlite@3.40.0/dist/sqlite3.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-db.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-examples.js"></script>
</head>
<body>

<pre class="employees">select
  rank() over w as "rank",
  name, department, salary
from employees
window w as (order by salary desc)
order by "rank", id;</pre>

<pre class="demo">select * from employees limit 10;</pre>

<sqlime-db name="employees" path="https://raw.githubusercontent.com/nalgeon/sqlime/main/employees.sql"></sqlime-db>
<sqlime-examples db="employees" selector="pre.employees" editable></sqlime-examples>

<sqlime-db name="emp2" path="https://raw.githubusercontent.com/nalgeon/sqlime/main/employees.sql"></sqlime-db>
<sqlime-examples db="emp2" selector="pre.demo" editable></sqlime-examples>

</body>
</html>
```

Now If I open this html file in my browser, I can run the 1st example, but getting errors in the second case,

![image](https://github.com/nalgeon/sqlime/assets/48413872/26cc7f1c-d409-4916-8e9e-997e23d2152e)

And errors from the google chrome devtools console,

![image](https://github.com/nalgeon/sqlime/assets/48413872/73868b81-e5c4-4008-9881-25f81fc2b71d)

But if I comment out one of the `sqlime-db`, `sqlime-examples` commands pair, everything works fine.

``` html
<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>sqlime</title>
	<script src="https://unpkg.com/@antonz/sqlite@3.40.0/dist/sqlite3.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-db.js"></script>
	<script src="https://unpkg.com/sqlime@0.1.2/dist/sqlime-examples.js"></script>
</head>
<body>

<pre class="employees">select
  rank() over w as "rank",
  name, department, salary
from employees
window w as (order by salary desc)
order by "rank", id;</pre>

<pre class="demo">select * from employees limit 10;</pre>

<!-- <sqlime-db name="employees" path="https://raw.githubusercontent.com/nalgeon/sqlime/main/employees.sql"></sqlime-db>
<sqlime-examples db="employees" selector="pre.employees" editable></sqlime-examples> -->

<sqlime-db name="emp2" path="https://raw.githubusercontent.com/nalgeon/sqlime/main/employees.sql"></sqlime-db>
<sqlime-examples db="emp2" selector="pre.demo" editable></sqlime-examples>

</body>
</html>
```

![image](https://github.com/nalgeon/sqlime/assets/48413872/eb5ff17c-f8f9-4866-93d6-b2170150ee29)

**Comment by nalgeon** [comment]
It's a bug, fixed in 9f33d6766a783da630994fd71bd5c36dda74739e. Please try version `0.1.3`.

**Comment by shafayetShafee** [comment]
Woah! All work fine now. 

Thanks for your prompt replies and quick help. Thank you so much.
